routes/messages.js

```diff
@@ -89,4 +89,61 @@ router.post("/", async (req, res) => {
   }
 });
 
+router.delete("/", async (req, res) => {
```

Comment:
This endpoint is kind of dangerous, will we ever use this in production context?